src/utils/constants/errorMessages.js

```diff
@@ -19,4 +19,33 @@ export const signupVerifyErrors = {
   notFound: 'No pending verification found'
 };
 
+export const userProfileErrors = {
+  undefinedFirstName: 'Firstname is required',
+  invalidFirstName: 'Firstname should be an Alphabet with no spaces',
+  undefinedLastName: 'Lastname is required',
+  invalidLastName: 'Lastname should be an Alphabet with no spaces',
+  undefinedEmail: 'Email is required',
+  undefinedPassword: 'Password is required',
+  invalidPassword: 'Password cannot be less than 8 characters',
+  invalidEmail: 'Enter a valid email address',
+  nonAndelanEmail: 'Email should be an andela email',
+  alphaNumericPassword: 'Password should be alphanumeric',
+  undefinedGender: 'Gender is required',
+  invalidGender: 'Gender should be either male or female',
+  undefinedBirthday: 'Birthday is required',
+  invalidBirthday: 'Birthday should be a date format mm-dd-yyyy',
+  undefinedPreferredLanguage: 'Preferred Language is required',
+  invalidPreferredLanguage: 'Preferred Language should be an Alphabet with no spaces',
+  undefinedCurrency: 'Currency is required',
+  invalidCurrency: 'Currency should be an Alphabet with no spaces',
```

Comment:
Technically, an Alphabet is one letter. I think we should change this message and other similar ones to something like `Currency should be a set of alphabets with no spaces`.